plugins/dfu/fu-dfu-device.c

```diff
@@ -75,7 +75,7 @@
 #include "fu-dfu-common.h"
 #include "fu-dfu-device.h"
 #include "fu-dfu-target-avr.h"
-#include "fu-dfu-target-private.h"
+#include "fu-dfu-target-private.h" //waive-pre-commit
```

Comment:
it makes me a bit uneasy seeing C++ comments for this -- we've used /**/ literally everywhere else in the codebase.